<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalTuner</class>
 <widget class="QDialog" name="SignalTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Group One</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,5">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>倍频:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_G1FactorSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_G1CurVal"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10,5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>分频:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_G1DividerSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_G1Mode"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>248</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>每个信号触发行数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_G1LineAcquire"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Group Two</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,10,5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>倍频:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_G2FactorSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_G2CurVal"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,10,5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>分频:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_G2DividerSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_G2Mode"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>248</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 0);
font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>每个信号触发行数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_G2LineAcquire"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_LoadBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 0);
color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>加载设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_SaveBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 136, 0);
color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>保存设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ApplyBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(82, 122, 61);
color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
